tests/integration/CapabilityTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Automattic\LegacyRedirector\Tests;
+
+use Automattic\LegacyRedirector\Capability;
+
+/**
+ * CapabilityTest class
+ */
+class CapabilityTest extends TestCase {
+
+	/**
+	 * We remove the Capability::MANAGE_REDIRECTS_CAPABILITY  before testing
+	 *
+	 * @beforeClass
+	 */
+	public static function beforeClassSetup() {
+		$capability = new Capability();
+		$capability->unregister();
+	}
```

Comment:
Is this needed? (It may be!)

As part of testing, we register the plugin as being activated, but that shouldn't trigger the `admin_init` hook which may do the cap registering.

If other test classes run first, I don't think it should be left in a caps-registered state.

If this method is scrapped, do the tests pass? If so, then it either it was redundant, or there's another thing to be tested.

Reply:
It seems it is, what happens is that after each test, the wp_options tests table retains it's roles option to the mentioned state, so I think it's best after each test to remove any set capabilities